README.rst

```diff
@@ -0,0 +1,81 @@
+PyYAML - The next generation YAML parser and emitter for Python.
+
+.. image:: https://travis-ci.org/yaml/pyyaml.svg?branch=master
+    :target: https://travis-ci.org/yaml/pyyaml
+
+.. image:: https://badge.fury.io/py/PyYAML.svg
+    :target: https://badge.fury.io/py/PyYAML
+
+YAML is a data serialization format designed for human readability
+and interaction with scripting languages.  PyYAML is a YAML parser
+and emitter for Python.
+
+PyYAML features a complete YAML 1.1 parser, Unicode support, pickle
+support, capable extension API, and sensible error messages.  PyYAML
+supports standard YAML tags and provides Python-specific tags that
+allow to represent an arbitrary Python object.
+
+PyYAML is applicable for a broad range of tasks from complex
+configuration files to object serialization and persistence.
+
+To install, type following:
+
+.. code-block:: shell
+
+    # recommended
+    pip install PyYAML
+    # or
+    python setup.py install
+
+By default, the setup.py script checks whether LibYAML is installed
+and if so, builds and installs LibYAML bindings.  To skip the check
+and force installation of LibYAML bindings, use the option ``--with-libyaml``:
+``python setup.py --with-libyaml install``.  To disable the check and
+skip building and installing LibYAML bindings, use ``--without-libyaml``:
+
+
+.. code-block:: shell
+
+    # recommended for pip >= v20.0
+    pip install pyyaml --global-option=--without-libyaml
```

Comment:
Best practice would be to use `python -m pip install` as detailed in https://snarky.ca/why-you-should-use-python-m-pip/.